rdr_service/resource/schemas/genomics.py

```diff
@@ -0,0 +1,222 @@
+#
+# This file is subject to the terms and conditions defined in the
+# file 'LICENSE', which is part of this source code package.
+#
+from marshmallow import validate
+
+from rdr_service.resource import Schema, SchemaMeta, fields
+from rdr_service.resource.constants import SchemaID
+
+from rdr_service.participant_enums import (
+    GenomicSetStatus,
+    GenomicSetMemberStatus,
+#    GenomicValidationFlag,
+    GenomicSubProcessStatus,
+    GenomicSubProcessResult,
+    GenomicJob,
+    GenomicWorkflowState
+)
+
+
+class GenomicSetSchema(Schema):
+
+    id = fields.Int32()
+    created = fields.DateTime()
+    modified = fields.DateTime()
+    genomic_set_name = fields.String(validate=validate.Length(max=80))
+    genomic_set_criteria = fields.String(validate=validate.Length(max=80))
+    genomic_set_version = fields.Int32()
+    genomic_set_file = fields.String(validate=validate.Length(max=250))
+    genomic_set_file_time = fields.DateTime()
+    genomic_set_status = fields.EnumString(enum=GenomicSetStatus)
+    genomic_set_status_id = fields.EnumInteger(enum=GenomicSetStatus)
+    validated_time = fields.DateTime()
+
+    class Meta:
+        """
+        schema_meta info declares how the schema and data is stored and organized in the Resource database tables.
+        """
+        ordered = True
+        resource_pk_field = 'id'
+        # SchemaMeta (unique type id, unique type name, type URI, resource pk field, nested schemas)
+        schema_meta = SchemaMeta(
+            type_uid=SchemaID.genomic_set.value,
+            type_name=SchemaID.genomic_set.name,
+            resource_uri='GenomicSet',
+            resource_pk_field='id'
+        )
+
+class GenomicSetMemberSchema(Schema):
+
+    id = fields.Int32()
+    created = fields.DateTime()
+    modified = fields.DateTime()
+    genomic_set_id = fields.Int32()
+    participant_id = fields.Int32()
+    ny_flag = fields.Int32()
+    sex_at_birth = fields.String(validate=validate.Length(max=20))
+    genome_type = fields.String(validate=validate.Length(max=80))
+    biobank_id = fields.String(validate=validate.Length(max=80))
+    package_id = fields.String(validate=validate.Length(max=80))
+    validation_status = fields.EnumString(enum=GenomicSetMemberStatus)
+    validation_status_id = fields.EnumInteger(enum=GenomicSetMemberStatus)
+    # validation_flags is an array of GenomicValidationFlag Enum values.
+    validation_flags = fields.String(validate=validate.Length(max=80))
+    validated_time = fields.DateTime()
+    sample_id = fields.String(validate=validate.Length(max=80))
```

Comment:
This field's datatype was changed to an integer in a recent PR: https://github.com/all-of-us/raw-data-repository/pull/1946